src/ui/src/channel/message.rs

```diff
@@ -9,5 +9,5 @@ pub fn message(message: impl Message) -> impl IntoElement {
     .gap_2()
     .p_2()
     .child(img(message.get_author().get_icon()).object_fit(gpui::ObjectFit::Fill).bg(rgb(0xFFFFFF)).rounded_full().w_12().h_12())
-    .child(div().flex().flex_col().child(message.get_author().get_display_name()).child(message.get_content()))
+    .child(div().flex().flex_col().child(message.get_author().get_display_name()).child(message.get_content()).child(message.get_images()))
```

Comment:
one other thing: this will conflict now that #11 is merged. Changing content should fix this though.